components/general/HowWeWork.tsx

```diff
@@ -1,6 +1,10 @@
 import InfoBlock from "./InfoBlock/InfoBlock";
 
-const HowWeWork = () => {
+type Props = {
+  isPhoneSize: boolean;
+};
+
+const HowWeWork = ({ isPhoneSize }: Props) => {
```

Comment:
Your task should be responsive update for ContactUs and LastestNews, so why you change this component?  I do not understand

Reply:
![image](https://user-images.githubusercontent.com/73293255/225636230-8a8e68c3-ec08-4951-a5b8-944c2e1d9664.png)